packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -202,19 +199,17 @@ function run(task) {
 
   // todo(flow->ts) add proper return type (added any, because empty object is inferred)
   function getOpts(self): any {
-    const newOpts = merge(
-      {
-        ast: true,
-        cwd: path.dirname(self.loc),
-        filename: self.loc,
-        filenameRelative: self.filename,
-        sourceFileName: self.filename,
-        sourceType: "script",
-        babelrc: false,
-        inputSourceMap: task.inputSourceMap || undefined,
-      },
-      opts,
-    );
+    const newOpts = {
+      ast: true,
+      cwd: path.dirname(self.loc),
+      filename: self.loc,
+      filenameRelative: self.filename,
+      sourceFileName: self.filename,
+      sourceType: "script",
+      babelrc: false,
+      inputSourceMap: task.inputSourceMap || undefined,
+      ...opts,
```

Comment:
It doesn't seem like we'd need to deep merge any of the options above so should be equivalent (or are all primitives)